Gemfile

```diff
@@ -6,6 +6,7 @@ git_source(:github) do |repo_name|
 end
 
 gem 'honeybadger', '~> 3.1'
+gem 'skylight'#,    '~>'
```

Comment:
Can you add a version or remove the `#`?

Reply:
Thanks, done.